DIRECTOR_HANDBOOK.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Foundation<br>Board of Directors Handbook
+
+Although the Node.js Foundation is a not-for-profit membership organization rather than a corporation owned by stockholders, the same legal rules and obligations apply to its Board of Directors as is the case with a publicly or privately held stock corporation.  Thus, while a member of the Board may have been nominated or even appointed by the Director’s employer as an exercise of a membership right, such a director must be mindful of a Director’s obligations under standing law and the Foundation's Certificate of Incorporation and By-laws.  The purpose of this document is to acquaint new and existing Directors of these obligations.
+
+It is the express policy of the Foundation to require that all its activities be conducted strictly in accordance with federal and state laws.  It is extremely important that all members of the Board of Directors be aware of the legal and ethical standards to which you are held and the types of activities to avoid in order to fulfill your duties as directors properly, as well as to avoid liability.
+
+These Governance Guidelines, Code of Conduct and Duties were prepared to provide you with a broad, summary overview of the standards under the law to which you will be held.  However, you should note that these governance guidelines and standards provide a general guide only; they are not intended to be a complete and definitive statement of all aspects of the law relating to the duties of Directors.  Any specific questions relating to your duties as a Director not addressed in these governance guidelines and standards should be referred to counsel for the Foundation or to your own legal counsel who has responsibility for considering your duties as a Director of the Foundation.
+
+## Board Governance
+
+The Board of Directors represents the interests of the Foundation’s Members.  The Board is the ultimate decision-making body of the Foundation except with respect to those matters reserved to the Members.  The business and affairs of the Foundation are managed by its Board.  Although the Board may delegate authority, the ultimate responsibility for the Foundation resides with the Board, which may not lawfully abdicate its duties.
+
+Directors need to be aware that the Foundation’s funds are dedicated to the purposes set forth in the Certificate of Incorporation, and may only be used in a manner consistent with the Foundation’s authorized tax-exempt purposes.
+
```

Comment:
It should also be noted that because of our tax exempt status spending must be authorized by the board and is done so by setting up budgets, and granting approval through board votes.  Board members will be called upon to cast their votes through out the year for the purpose of spending funds.